Agents/HeatEmissionAgent/README.md

```diff
@@ -1,54 +1,104 @@
 # Heat Emission Agent
- This agent is to return the heat emission information of different buildings/objects presented in a given area. Therefore, the heat emission data can be automatically assigned to corresponding buildings within a specific area. This is of interest to the Cooling Singapore 2.0 Project. 
- 
- It demonstrates firstly the feasibility of a cross-domain query and secondly the evaluation of the heat emission data in terms of emission values and respective coordinates within a bounding box in Jurong Island. To achieve this, it consists of four parts. First, we obtain all the chemical plants, plant items, IRIs and CO2 emission via query in "jibusinessunits"; Second, for a particular chemical plant, its fuel CEI and thermal efficiency are queried; subsequently, all the heat emission coordinates are evaluated via query in "jriEPSG24500"; finally, the heat emission values are calculated with CO2 emission, CEI and efficiency and assigned to the emission coordinates, after a filter based on a boundary area specified.The heat emission data are saved into the TheWorldAvatar blazegraph.
 
-# Example Heat Emission Data
-Heat Emission Data is returned to the response body in form of a JSON Object which consist of key-value pair. The JSONObject has the 
-key:"result", which contains a JSONArray containing JSONObjects. Each of these individual JSONObjects found within the JSONArray
-provide the heat emission coordinates and the corresponding heat emission value in the unit of MW.
+The heat emission agent calculates heat emissions of data centres and various types of factories. The properties of these industrial facilities that are required to calculate the heat emissions need to be stored in PostgreSQL tables as explained below. 
```

Comment:
Are the terms 'factories' and 'industrial facilities' interchangeable? According to the Miro board it seems they are different - if they are indeed different then revise the above sentences

Reply:
The classes 'Factory' and 'DataCentre' are subclasses of 'IndustrialFacility'.